pandas/tests/window/test_rolling.py

```diff
@@ -1203,6 +1203,9 @@ def test_rolling_var_numerical_issues(func, third_value, values):
     result = getattr(ds.rolling(2), func)()
     expected = Series([np.nan] + values)
     tm.assert_series_equal(result, expected)
+    # GH 42064
+    # new `roll_var` will output 0.0 correctly
+    tm.assert_series_equal(result == 0, expected == 0)
```

Comment:
this test doesn't make sense. is the expected wrong?

Reply:
Apparently with `check_exact=True` (default is False & checks within a tolerance), these tests fail the 32 bit build.

https://github.com/pandas-dev/pandas/pull/46431#discussion_r836983428=

I think this is okay since it specifically asserts the 0's

Reply:
> I think this is okay since it specifically asserts the 0's
Yeah, though it looks redundant.